docs/public/upgrading/backend/upgrade_zonemaster_backend_ver_11.2.0.md

```diff
@@ -0,0 +1,17 @@
+# Upgrade to 11.2.0
+
+## Upgrading the database
+
+If your Zonemaster database was created by a Zonemaster-Backend version smaller
+than v11.2.0, and not upgraded, use the following instructions.
+
+> You may need to run these command with root privileges.
+
+### Upgrade database
+
+The following script fixes existing untranslated entries that were missed between 
+Backend version 11.0 (release v2023.2) and this version (release v2024.1).
```

Comment:
In release v2023.2 the Zonemaster-Backend version is v11.1.0. Better to spell out "Zonemaster-Backend" in all places.

To be more explicit and easier for the reader:

"Zonemaster-Backend version v11.1.0 (release v2023.2) and this version (release v2024.1)"